spec/light-client/supervisor/supervisor_001_draft.md

```diff
@@ -463,69 +465,95 @@ func Sequential-Supervisor (initdata LCInitData) (Error) {
 #### Initialization
 
 The light client is based on subjective initialization. It has to
-trust the initial data given to it by the user. It cannot do any
-detection of attack. So either upon initialization we obtain a
-lightblock and just initialize the lightstore with it. Or in case of a
-genesis file, we download, verify, and cross-check the first block, to
-initialize the lightstore with this first block. The reason is that
-we want to maintain [LCV-INV-TP.1] from the beginning.
-
-> If the lightclient is initialized with a lightblock, one might think
-> it may increase trust, when one cross-checks the initial light
-> block. However, if a peer provides a conflicting
-> lightblock, the question is to distinguish the case of a
-> [bogus](https://informal.systems) block (upon which operation should proceed) from a
-> [light client attack](https://informal.systems) (upon which operation should stop). In
-> case of a bogus block, the lightclient might be forced to do
-> backwards verification until the blocks are out of the trusting
-> period, to make sure no previous validator set could have generated
-> the bogus block, which effectively opens up a DoS attack on the lightclient
-> without adding effective robustness.
+trust the initial data given to it by the user. It cannot perform any
+detection of an attack yet instead requires an initial point of trust.
+There are three forms of initial data which are used to obtain the 
+first trusted block:
+
+- A trusted block from a prior initialization
+- A trusted height and hash
+- A genesis file
+
+The golang light client implementation checks this initial data in that
+order; first attempting to find a trusted block from the trusted store,
+then acquiring a light block from the primary at the trusted height and matching
+the hash, or finally checking for a genesis file to verify the initial header.
+
+The light client doesn't need to check if the trusted block is within the
+trusted period because it already trusts it, however, if the light block is
+outside the trust period, there is a higher chance the light client won't be
+able to verify anything.
```

Comment:
If the trusted block is outside of the "trust period" (i.e. all of its validators have already unbonded), can it be trusted at all? Light client would be subject to a long range attack in that case, correct?